Content.Shared/Roles/MindRoleComponent.cs

```diff
@@ -0,0 +1,53 @@
+using Content.Shared.Mind;
+using JetBrains.Annotations;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Roles;
+
+/// <summary>
+/// This holds data for, and indicates, a Mind Role entity
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class MindRoleComponent : BaseMindRoleComponent
+{
+    /// <summary>
+    ///     Marks this Mind Role as Antagonist
+    ///     A single antag Mind Role is enough to make the owner mind count as Antagonist.
+    /// </summary>
+    [DataField]
+    public bool Antag { get; set; } = false;
```

Comment:
Couldn't this just be `=> AntagPrototype is not null` instead of storing a bool? Or is it possible for a role to be antag but not have a prototype?

Reply:
There are Ghostroles that are antag but don't have an antagprototype, such as dragons and carps